dmoj/executors/java_executor.py

```diff
@@ -268,7 +271,7 @@ def get_compile_args(self) -> List[str]:
         compiler = self.get_compiler()
         assert compiler is not None
         assert self._code is not None
-        return [compiler, '-Xlint', '-encoding', 'UTF-8', self._code]
+        return [compiler, '-encoding', 'UTF-8', self._code]
```

Comment:
Why did we kill `-Xlint`?

Reply:
Neither `JAVA.py` nor `JAVA8.py` use `-Xlint` from what I can tell.